.circleci/config.yml

```diff
@@ -673,7 +672,7 @@ jobs:
     steps:
       - attach_workspace:
           at: ~/project
-      - run: docker run --entrypoint "./run_litmus_tests_oss.sh" -e TEST_LIST=tests_lists/gateway_api_tests.list -e INFLUXPATH=/Litmus/result/dist/influx_linux_amd64/influx -e BINARYPATH=/Litmus/result/dist/influxd_linux_amd64/influxd -e BOLTPATH=/tmp/influxd_test/influxd.bolt -e ENGINEPATH=/tmp/influxd_test --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/Litmus/result quay.io/influxdb/litmus:latest
+      - run: docker run --entrypoint "./run_litmus_tests_oss.sh" -e TEST_LIST=tests_lists/gateway_api_tests.list -e BINARYPATH=/Litmus/result/dist/influxd_linux_amd64/influxd -e BOLTPATH=/tmp/influxd_test/influxd.bolt -e ENGINEPATH=/tmp/influxd_test --net host -v /var/run/docker.sock:/var/run/docker.sock -v ~/project:/Litmus/result quay.io/influxdb/litmus:latest
```

Comment:
Did this just never need to the `INFLUXPATH` anyway?

Reply:
Searching in the repo shows it might be used for 1 `restore` test. I pushed a commit to see if that test is running on merges to `master`; if so, I'll look at it and see if it can be deleted (we have a pretty comprehensive backup/restore integration test in `influxdb` already)

Reply:
For future investigators: the test was skipped, I deleted it + the logic for tracking the influx CLI in influxdata/litmus-2.0#13 and influxdata/litmus-2.0#14